docs/terms/data_docs.md

```diff
@@ -1,14 +1,100 @@
+
+<CreateHeader/>
+
+You can view your saved Expectation Suites in Data Docs.  
+
+<ValidateHeader/>
+
+Saved Validation Results will be displayed in any Data Docs site that is configured to show them.  If you build your Data Docs from the Data Context, the process will render Data Docs for all of your Validation Results.  Alternatively, you can use the `UpdateDataDocsAction` Action in a Checkpoint's `action_list` to update your Data Docs with just the Validation Results generated by that checkpoint.
+
+## Features
+
+### Readability
+
+Data Docs provide a clean, human-readable way to view your Expectation Suites and Validation Results without you having to manually parse their stored values and configurations.  You can also [add comments to your Expectations that will be displayed in your Data Docs](../guides/expectations/advanced/how_to_add_comments_to_expectations_and_display_them_in_data_docs.md), if you feel they need further explanation.
+
+### Versatility
+
+There are multiple use cases for displaying information in your Data Docs.  Three common ones are:
+
+1. Visualize all Great Expectations artifacts from the local repository of a project as HTML: Expectation Suites,
+   Validation Results and profiling results.
+1. Maintain a "shared source of truth" for a team working on a data project. Such documentation renders all the
+   artifacts committed in the source control system (Expectation Suites and profiling results) and a continuously
+   updating data quality report, built from a chronological list of validations by run id.
+1. Share a spec of a dataset with a client or a partner. This is similar to API documentation in software development.
+   This documentation would include profiling results of the dataset to give the reader a quick way to grasp what the
+   data looks like, and one or more Expectation Suites that encode what is expected from the data to be considered
+   valid.
+
+To support these (and possibly other) use cases Great Expectations has a concept of "data documentation site". Multiple sites can be configured inside a project, each suitable for a particular data documentation use case.
+
+### Customization
+
+The HTML documentation generated by Great Expectations Data Docs is fully customizable. If you would like to customize the look and feel of these pages or create your own, see our guide on [how to create Renderers for Custom Expectations](../guides/expectations/advanced/how_to_create_renderers_for_custom_expectations.md).
+
+## API basics
+
+### How to access
+
+Data Docs are rendered as HTML files.  As such, you can open them with any browser.
+
+### How to create
+
+If your Data Docs have not yet been rendered, you can create them from your Data Context.
+
+From the root folder of your Data Context, you can build your Data Docs with the CLI command:
```

Comment:
Might we want to say "the root folder of your project"? I could see users thinking the root folder of `great_expectations/`, which would not work as intended.

Reply:
I updated it to say the root folder of their project, and specified that it was where their Data Context would have been initialized from.